WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -54,15 +53,7 @@ public async Task StartAsync(Wallet wallet, CancellationToken cancellationToken)
 
 	public async Task StartAutomaticallyAsync(Wallet wallet, CancellationToken cancellationToken)
 	{
-		using var combinedTokenSource = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, AbortCancellationTokenSource.Token);
-		var combinedToken = combinedTokenSource.Token;
```

Comment:
This gets called with `CancellationToken.None` from `CoinJoinStateViewModel` [here](https://github.com/zkSNACKs/WalletWasabi/blob/3fbdf348aff43ed1a85209f8d4f31af0a4c2bb82/WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs#L285).
Won't this cause us problems?

Reply:
With this PR the token is only used for Channel Writer which is usually quick and returns immediately. In this scenario, the channel is not heavily used so it is no need to add a timeout.